src/components/shared/TreeView.js

```diff
@@ -748,6 +748,7 @@ export class TreeView<DisplayData: Object> extends React.PureComponent<
         <TreeViewHeader fixedColumns={fixedColumns} mainColumn={mainColumn} />
         <ContextMenuTrigger
           id={contextMenuId}
+          disable={!contextMenuId}
```

Comment:
I could make it so `ContextMenuTrigger` isn't used at all in this case, but I found this simpler.